README.md

```diff
@@ -1,2 +1,50 @@
 # TSC
-The AdoptOpenJDK Technical Steering Comittee 
+The AdoptOpenJDK Technical Steering Comittee is the technical governing body of the AdoptOpenJDK organisation. It admits and oversees all top-level Projects in the AdoptOpenJDK organisation.
+
+### Dependent Projects
+These projects are located in the following repositories:
+
+* https://github.com/AdoptOpenJDK/TSC
+* https://github.com/AdoptOpenJDK/openjdk-build
+* https://github.com/AdoptOpenJDK/openjdk-website
+* https://github.com/AdoptOpenJDK/openjdk-tests
+* https://github.com/AdoptOpenJDK/openjdk-api
+* https://github.com/AdoptOpenJDK/openjdk-nightly
+* https://github.com/AdoptOpenJDK/openjdk-releases
+* https://github.com/AdoptOpenJDK/openjdk-jdk8u
+* https://github.com/AdoptOpenJDK/openjdk-jdk9
+* https://github.com/AdoptOpenJDK/openjdk-infrastructure
+* https://github.com/AdoptOpenJDK/openjdk-website-backend
+* https://github.com/AdoptOpenJDK/secrets _private_
+
+### List of TSC Responsibilities
+The TSC exercises autonomy in setting up and maintaining procedures, policies, and management and administrative structures as it deems appropriate for the maintenance and operation of these projects and resources.
+
+Included in the responsibilities of the TSC are:
+
+* Managing code and documentation creation and changes for the listed projects and resources
+* Meeting monthly to discuss progress and other TSC issues
+* Setting and maintaining standards covering contributions of code, documentation and other materials
+* Managing code and binary releases: types, schedules, frequency, delivery mechanisms
+* Creating new repositories and projects under the _AdoptOpenJDK_ GitHub organisation as required
+* Setting overall technical direction for the AdoptOpenJDK organisation, including high-level goals and low-level specifics regarding features and functionality
+* Setting and maintaining appropriate standards for community discourse via the various mediums under TSC control
+
+## TSC Members
+
+TSC members can nominate new members at any time. Candidates for membership tend to be people
+who have a competancy for community management and a high tolerance and patience for process
+minutiae as the TSC delegates most of its responsibilities to other teams.
+
+Every Dependant Project not currently incubating can appoint someone to the TSC who they elect
+at their own discretion.
+
+### Current Members
+| Avatar | Information |
+|---|:---|
+|   |    |
+
+## Rules for TSC Membership
+* No single company can occupy more than 1/3 seats on the TSC
+* The appointing of TSC members occurs on an annual basis and is based on a meritocracy. 
```

Comment:
Is the full card appointed each year, or people serve a one-year term?  Would be good to have some continuity of experience.